app/src/renderer/components/common/ActionModal.vue

```diff
@@ -380,4 +380,15 @@ export default {
   transform: translateX(2rem);
   opacity: 0;
 }
+
+@media screen and (max-width: 1024px) {
```

Comment:
I don't remember the outcome of the CSS discussion [mobile first min-width VS desktop first max-width], I just bring it up as seems that max-width was the winner, is it right?

Currently we have a lot of both in the project.

Reply:
i remember the conversation but not the outcome. there are a few min-widths left from tm-ui but otherwise i've been using max-width which seems to be working.